install_dependencies.R

```diff
@@ -1,4 +1,4 @@
-install.packages(c('testthat', 'vdiffr', 'spelling','covr','remotes'), repos = 'http://cran.us.r-project.org')
+install.packages(c('testthat', 'vdiffr', 'spelling','covr','remotes'), repos = 'http://cran.us.r-project.org', type='win.binary')
```

Comment:
Force **binary** package installation for the manually installed packages as well